.pre-commit-config.yaml

```diff
@@ -28,7 +28,6 @@ repos:
     rev: 6.0.0
     hooks:
       - id: flake8
-        language_version: python39
```

Comment:
FYI: This was needed to avoid an error in the `pre-commit`-generated virtual environments.

Reply:
Thanks for mentioning this! I was in a rush and forgot to break it out into a separate commit. (Probably should be a separate PR.)

Adding `python39` breaks pre-commit locally for me, since I don't have a `python39` executable. I think there must be some underlying issue. Could you share the error which led you to introduce this?

Reply:
I believe it was this issue: https://github.com/python/importlib_metadata/issues/406.

Reply:
I've add the same issue as @maresb locally.

Reply:
@rlouf, we resolved it in https://github.com/aesara-devs/aesara/pull/1374 :smiley:

Reply:
And after rebase the change to `.pre-commit-config.yml` is gone from this branch.